packages/rum-core/src/error-logging/error-logging.js

```diff
@@ -171,22 +171,24 @@ class ErrorLogging {
   }
 
   logPromiseEvent(promiseRejectionEvent) {
-    const prefix = 'Unhandled promise rejection: '
-    const { reason } = promiseRejectionEvent
+    const prefix = 'Uncaught (in promise) '
```

Comment:
Not so much opposed to naming the message in this format. Chrome and Firefox follows the new way, while Safari does with old format.